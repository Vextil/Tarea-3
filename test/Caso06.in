# ColaPrioridad
InsertarCP FIN
ElementosCP 5 1
ExtraerCP
InsertarCP cuarenta 40 cuarentaycinco 40 cuarentaytres 40 cuarenta 40 FIN
ElementosCP 5 1
ElementosCP 40 1
ImprimirLista 1
ExtraerCP
ElementosCP 40 1
ElementosCP 40 1
ImprimirLista 1
InsertarCP veinte 20 FIN
ElementosCP 5 1
ElementosCP 20 1
ElementosCP 40 2
ImprimirLista 1
ImprimirLista 2
InsertarCP veintiuno 20 FIN
ExtraerCP
ImprimirLista 1
ElementosCP 20 1
ImprimirLista 1
InsertarCP diez 10 FIN
ExtraerCP
InsertarCP treintaidos 30 treinta 30 FIN
ExtraerCP
ExtraerCP
InsertarCP treintaicinco 35 veiticinco 25  treintaiocho 38 quince 15  FIN
ExtraerCP
InsertarCP treintaicinco2 35 treintaiuno 31 noventainueve 99 doce 12 FIN
ElementosCP 20 1
ElementosCP 12 2
ElementosCP 40 1
ImprimirLista 2
ImprimirLista 1
ExtraerCP
ExtraerCP
ExtraerCP
ExtraerCP
ExtraerCP
ExtraerCP
ElementosCP 12 1
ExtraerCP
ExtraerCP
ExtraerCP
ElementosCP 40 1
ImprimirLista 1
ExtraerCP
ExtraerCP
ExtraerCP
Fin
InsertarCP txt 40 sed 20 fgy 5 lop 95 vfg 25 fhg 40 gjg 40 dgg 5 hkhjk 95 FIN
ElementosCP 5 1
ImprimirLista 1
ExtraerCP
ElementosCP 50 2
ElementosCP 95 3
ImprimirLista 3
Fin

       12
    25      31 
  30 38   35  99
40