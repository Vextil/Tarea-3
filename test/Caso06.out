1># ColaPrioridad
2>OK
3>No hay elementos con esa prioridad.
4>La cola de prioridad estaba vacia. 
5>OK
6>No hay elementos con esa prioridad.
7>OK
8>NO ordenada. cuarenta cuarentaycinco cuarentaytres cuarenta 
9>cuarenta - 40
10>OK
11>OK
12>NO ordenada. cuarentaycinco cuarentaytres cuarenta 
13>OK
14>No hay elementos con esa prioridad.
15>OK
16>OK
17>Ordenada. veinte 
18>NO ordenada. cuarentaycinco cuarentaytres cuarenta 
19>OK
20>veinte - 20
21>Ordenada. veinte 
22>OK
23>Ordenada. veintiuno 
24>OK
25>diez - 10
26>OK
27>veintiuno - 20
28>treintaidos - 30
29>OK
30>quince - 15
31>OK
32>No hay elementos con esa prioridad.
33>OK
34>OK
35>Ordenada. doce 
36>NO ordenada. cuarentaycinco cuarentaytres cuarenta 
37>doce - 12
38>veiticinco - 25
39>treinta - 30
40>treintaiuno - 31
41>treintaicinco - 35
42>treintaicinco2 - 35
43>No hay elementos con esa prioridad.
44>treintaiocho - 38
45>cuarentaycinco - 40
46>cuarentaytres - 40
47>OK
48>Ordenada. cuarenta 
49>cuarenta - 40
50>noventainueve - 99
51>La cola de prioridad estaba vacia. 
52>Fin
